util/megaquit.m

```diff
@@ -0,0 +1,4 @@
+%NOTEST
```

Comment:
alright, i'll take it out (but probably keep using it locally). I actually find this really useful, since editing c++ files requires frequent restart of matlab, and I can't figure out how to get matlab to stop asking me if I want to save my simulink blocks (or just sometimes refusing to quit because a simulation is still running)